tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{310,311,312}-test
+    py{310,311,312,313,314}-test
```

Comment:
```suggestion
    py{311,312,313,314}-test
```

Reply:
We're actually right on the button for dropping 3.11 according to SPEC 0?

Reply:
Oh lmao. Sure

Reply:
I'll make more suggestions